zebra/zebra_vxlan.c

```diff
@@ -3799,12 +3799,13 @@ void zebra_vxlan_remote_macip_del(ZAPI_HANDLER_ARGS)
 	char buf1[INET6_ADDRSTRLEN];
 
 	memset(&macaddr, 0, sizeof(struct ethaddr));
-	memset(&ip, 0, sizeof(struct ipaddr));
 	memset(&vtep_ip, 0, sizeof(struct in_addr));
 
 	s = msg;
 
 	while (l < hdr->length) {
+		memset(&ip, 0, sizeof(struct ipaddr));
```

Comment:
why is this the only one that needs to be memset per loop iteration?

And if it really is the only one, can we just init the other two above with `= {}`  to make it clear.